features/kafka/producer/src/test/java/org/opennms/features/kafka/producer/KafkaForwarderBlueprintTest.java

```diff
@@ -43,34 +46,35 @@
  *
  * @author jwhite
  */
-@Ignore
+//@Ignore
```

Comment:
Does this test work reliably now? There's still a comment in the header here saying otherwise.